js/main.js

```diff
@@ -0,0 +1,119 @@
+/* global data */
+/* exported data */
+
+// DOM elements
+
+var $startPage = document.querySelector('.start-page');
+var $gamePage = document.querySelector('.game-page');
+
+var $startGameBtn = document.querySelector('.start-game-btn');
+
+var $dealCardsBtn = document.querySelector('.deal-cards-btn');
+
+var $playerHand = document.querySelector('.player-hand');
+
+var $dealerHand = document.querySelector('.dealer-hand');
+
+// global variables
+
+var drawnCards = null;
+
+// XHR
+
+function getDecks(numOfDecks) {
+  var xhr = new XMLHttpRequest();
+
+  xhr.open('GET', 'http://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=' + numOfDecks);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    data.currentDeckId = xhr.response.deck_id;
+  });
+
+  xhr.send();
+}
+
+function dealCards(player, numOfCards) {
+  var xhr = new XMLHttpRequest();
+
+  xhr.open('GET', 'http://deckofcardsapi.com/api/deck/' + data.currentDeckId + '/draw/?count=' + numOfCards);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    drawnCards = xhr.response;
+
+    for (var i = 0; i < drawnCards.cards.length; i++) {
+      player.hand.push(drawnCards.cards[i]);
+    }
+
+    if (data.dealer.hand.length === 0) {
+      dealCards(data.dealer, 2);
+    }
```

Comment:
Instead of making two requests, can you just make a request for 4 cards and deal them to the dealer and the player?